### DISCUSSION TITLE
How to jump to next version control change?

### DISCUSSION THREAD

**Discussion by thomasgtaylor** [zed-industries/zed#11173] [Feature Requests]
Hello! I may have simply missed the configuration option in the documentation, but is there a way to jump to next version control change / modification?

For example, I use the `editor::GoToDiagnostic` and `editor::GoToPrevDiagnostic` when moving between errors:

```json
{
      "f8": "editor::GoToDiagnostic",
      "shift-f8": "editor::GoToPrevDiagnostic",
}
```

Is there a similar one for version control?

**Comment by bennetbo** [comment]
Yeah there is `editor::GoToHunk`/`editor::GoToPrevHunk` which lets you jump to additions/modifications (default keybindings: `cmd-f8`/`cmd-shift-f8`)

**Reply by thomasgtaylor** [reply]
Thank you! I got it working!

**Comment by thomasgtaylor** [comment]
For anyone looking at this now, version [0.177.7](https://github.com/zed-industries/zed/releases/tag/v0.177.7) changed the action to `editor::GoToPreviousHunk` instead of `editor::GoToPrevHunk`.
